appstudio-utils/util-scripts/verify-attestation-with-policy.sh

```diff
@@ -45,3 +45,7 @@ title Violations
 
 title "Passed?"
 ./ec-pass-fail.sh "${output}" | tee "${passed}"
+
+if [[ "${STRICT_POLICY:-'1'}" = "1" ]]; then
```

Comment:
I know it's the same, but maybe `==` is kinder to the reader here..

Reply:
Perhaps it's clear enough, but I would consider a comment here, e.g

    # Return a non-zero exit code if the policy didn't pass

Reply:
Again I forgot about set -e, maybe:

     # Cause task to fail

Reply:
`==` is a nice idea. I never know how to properly do conditionals in bash. I verified that using `==` works in different cases. Fixed.